src/domain/signout/result/signout.response.result.ts

```diff
@@ -0,0 +1,6 @@
+export class SignoutResponseResult {
+  is_updated: boolean;
+  constructor(is_updated: boolean) {
+    this.is_updated = is_updated;
+  }
+}
```

Comment:
저희 내부적으로 계층 간에 쓰는 dto의 경우에는 이미 검증된 내용들이 오가는 곳입니다. 그러니 구체 클래스보다는 interface를 쓰는게 어떨까요??
```suggestion
export interface SignoutResponseResult {
  is_updated: boolean;
}
```